backend/user-service/psql.ts

```diff
@@ -0,0 +1,44 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const pool = new Pool({
+    host     : process.env.PSQL_HOSTNAME,
+    database : process.env.PSQL_DATABASE,
+    user     : process.env.PSQL_USERNAME,
+    password : process.env.PSQL_PASSWORD,
+    port     : parseInt(process.env.PSQL_PORT || '5432')
+});
+
+const createUserTableQueryIfNotExist = `
+    DO $$ 
+    BEGIN
+        IF NOT EXISTS (
+          SELECT 1
+          FROM information_schema.tables
+          WHERE table_name = 'users'
+        ) THEN
+          CREATE TABLE users (
+              user_id SERIAL PRIMARY KEY,
```

Comment:
And just for naming consistency for other schema definition, do you mind renaming this to `id` instead?

Reply:
sure, ill rename this to uid

Reply:
Resolved